File: Program form 1 to 10/qno.8/Main.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        double fahrenheit,celsius;
        System.out.print("Enter the temperature in fahrenheit:");
        fahrenheit=sc.nextDouble();
        celsius=(5.0/9.0)*(fahrenheit-32);
        System.out.println("The temperature in celsius is:" +celsius);
        sc.close();
        
    }
}

File: Program form 1 to 10/qno.8/main.c
#include<stdio.h>
int main()
{
    float celsius,fahrenheit;
    printf("Enter the temperature in fahrenhiet:");
    scanf("%f",&fahrenheit);
    celsius=(fahrenheit-32)*5/9;
    printf("%.2f fahrenheit=%.2f celsius\n",fahrenheit,celsius);
    return 0;
}